src/launchpad/service.py

```diff
@@ -519,6 +526,28 @@ def _upload_results(
         else:
             logger.info(f"Successfully uploaded analysis results for artifact {artifact_id}")
 
+    def _send_final_update_with_timestamps(
```

Comment:
Why do we need to make an API call at the end instead of just including the field in our JSON result we upload to the monolith? Like our `BaseAnalysisResults` already has an `analysis_duration` value in it, and are reading the file in `assemble_preprod_artifact_size_analysis`.

Reply:
to be clear we do need to send the `dequeued_at` field as part of the app info request, but I don't think we need this second request.